### PR TITLE
GetJobs (Query, Test, Endpoint)

### PR DESCRIPTION
**Pull request by hashemt**
(Query, Test, Endpoint)
Relates #40

### CODE REVIEW

File: server/tests/queries/getJobQuery.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const dbBuild = require('../../db/config/build.js');
+const insertFakeData = require('../../db/config/insertFakeData');
+const { getJobs } = require('../../db/queries/getJobs');
+
+exports.getJobsQuery = test('testing for the length of jobs information from database', (t) => {
+  const expected = 11;
+  dbBuild()
+    .then(() => insertFakeData())
+    .then(() => getJobs(2))
+    .then((jobs) => t.deepEqual(Object.keys(jobs.rows[0]).length, expected, 'should be the same value which shows the number of keys'))
```
Comment (fares98) [inline server/tests/queries/getJobQuery.js]: async-await instead of promise

File: server/tests/queries/getJobQuery.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const dbBuild = require('../../db/config/build.js');
+const insertFakeData = require('../../db/config/insertFakeData');
+const { getJobs } = require('../../db/queries/getJobs');
+
+exports.getJobsQuery = test('testing for the length of jobs information from database', async (t) => {
+  const expectedKeys = 11;
+  await dbBuild();
+  await insertFakeData();
+  const jobs = await getJobs(2);
```
Comment (AbdallahAmmar96) [inline server/tests/queries/getJobQuery.js]: Wher is the try and catch

File: server/tests/queries/getJob.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const test = require('tape');
+
+const dbBuild = require('../../db/config/build.js');
+const insertFakeData = require('../../db/config/insertFakeData');
+const { getJobs } = require('../../db/queries/getJobs');
+
+exports.getJobsQuery = test('testing for the length of jobs information from database', async (t) => {
+  try {
+    const expectedKeys = 11;
```
Comment (fares98) [inline server/tests/queries/getJob.js]: You can make it dynamic not constant

File: server/tests/routes/index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 const tape = require('tape');
 
+require('./getJob');
+
 tape('Simple pass test', (t) => {
```
Comment (AbdallahAmmar96) [inline server/tests/routes/index.js]: delete this test 1 = 1

File: server/tests/routes/getJob.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  await insertFakeData();
+  supertest(app)
+    .get('/api/v1/jobs')
+    .set('Cookie', ['jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwiZW1haWwiOiJmYWRpMTIzQGdtYWlsIiwidXNlcm5hbWUiOiJmYWRpIiwiaXNIYW5keW1hbiI6ZmFsc2UsImlhdCI6MTU2OTI2Nzg0OH0.wnVIwaHXQ6jpPmsx4FEnGgLRWanBINHvPpiaqBLA1YE'])
+    .expect(200)
+    .end((err, res) => {
+      if (err) {
+        t.error(err);
+      } else {
+        const jonObjectLength = Object.keys(res.body.data[0]).length;
```
Comment (AbdallahAmmar96) [inline server/tests/routes/getJob.js]:
```suggestion
        const jobObjectLength = Object.keys(res.body.data[0]).length;
```

File: server/tests/routes/getJob.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        t.equal(jonObjectLength, 11, 'should be equal the length of object keys which in database');
```
Comment (AbdallahAmmar96) [inline server/tests/routes/getJob.js]:
```suggestion
        t.equal(jobObjectLength, 11, 'should be equal the length of object keys which in database');
```

File: server/tests/queries/getJob.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    await dbBuild();
+    await insertFakeData();
+    const jobs = await getJobs(2);
+    t.deepEqual(Object.keys(jobs.rows[0]).length, expectedKeys, 'should be the same value which shows the number of keys');
+    t.end();
+  } catch (err) {
+    t.error(err);
+    t.end();
```
Comment (AbdallahAmmar96) [inline server/tests/queries/getJob.js]:
```suggestion
    
```
no need for t.end

**Pull request merged**